values/jobs/keycloak.gotmpl

```diff
@@ -12,10 +12,10 @@
 {{- $cm := $c | get "cert-manager" -}}
 {{- $k := $c | get "keycloak" dict }}
 {{- $skipVerify := eq ($cm | get "stage") "staging" }}
-tasks:
-
+jobs:
   keycloak:
     type: job
+    deployAlways: true
```

Comment:
Btw, this only needs to run "OnChange".

Reply:
That could be, but the original logic was "Always" (or equivalent).